bpfassets/libbpf/src/kepler.bpf.c

```diff
@@ -247,7 +248,8 @@ int kepler_trace(struct sched_switch_args *ctx)
         process_metrics_t new_process = {};
         new_process.pid = new_pid;
         //new_process.cgroup_id = cgroup_id;
-        bpf_get_current_comm(&new_process.comm, sizeof(new_process.comm));
```

Comment:
https://github.com/iovisor/bcc/blob/master/docs/reference_guide.md#6-bpf_get_current_comm

is it because we should use cur_pid instead of new_pid ?

Reply:
The change of `new_pid` from `cur_pid` is stated in this PR https://github.com/sustainable-computing-io/kepler/pull/819/files. @rootfs validates that we should use `new_pid` instead of `cur_pid` for libbpf. Need confirmation.

Reply:
ok, thanks for the link , so new_pid is the right one